# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gpu-alloc/src/heap.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gpu-alloc/src/heap.rs b/third_party/rust/gpu-alloc/src/heap.rs
--- a/third_party/rust/gpu-alloc/src/heap.rs
+++ b/third_party/rust/gpu-alloc/src/heap.rs
@@ -1,32 +1,32 @@
-#[derive(Debug)]
-pub(crate) struct Heap {
-    size: u64,
-    used: u64,
-    allocated: u128,
-    deallocated: u128,
-}
-
-impl Heap {
-    pub(crate) fn new(size: u64) -> Self {
-        Heap {
-            size,
-            used: 0,
-            allocated: 0,
-            deallocated: 0,
-        }
-    }
-
-    pub(crate) fn size(&mut self) -> u64 {
-        self.size
-    }
-
-    pub(crate) fn alloc(&mut self, size: u64) {
-        self.used += size;
-        self.allocated += u128::from(size);
-    }
-
-    pub(crate) fn dealloc(&mut self, size: u64) {
-        self.used -= size;
-        self.deallocated += u128::from(size);
-    }
-}
+#[derive(Debug)]
+pub(crate) struct Heap {
+    size: u64,
+    used: u64,
+    allocated: u128,
+    deallocated: u128,
+}
+
+impl Heap {
+    pub(crate) fn new(size: u64) -> Self {
+        Heap {
+            size,
+            used: 0,
+            allocated: 0,
+            deallocated: 0,
+        }
+    }
+
+    pub(crate) fn size(&mut self) -> u64 {
+        self.size
+    }
+
+    pub(crate) fn alloc(&mut self, size: u64) {
+        self.used += size;
+        self.allocated += u128::from(size);
+    }
+
+    pub(crate) fn dealloc(&mut self, size: u64) {
+        self.used -= size;
+        self.deallocated += u128::from(size);
+    }
+}